skaha/src/main/java/org/opencadc/skaha/session/SessionAction.java

```diff
@@ -374,10 +374,15 @@ public void streamPodLogs(String forUserID, String sessionID, OutputStream out)
     public Session getSession(String forUserID, String sessionID) throws Exception {
```

Comment:
Is getSession used elsewhere?  I just want to confirm that this won't change the behaviour of the skaha API call to /skaha/session (ie list sessions).

Reply:
SessionAction.getSession(...) is used by GetAction.getSingleSession(...) only so far.